File: acm_statistic_backend/api/api/DTO/performance/aeroport/RedevanceAeroportGlobalDTO.cs
namespace api.DTO.performance.aeroport
{
    public class RedevanceAeroportGlobalDTO
    {
        public decimal  Montant { get; set; }
        public string Aeroport { get; set; }
        public int Passagers {  get; set; }
        public int Vols { get; set; }
    }
}

File: acm_statistic_frontend/Interface Tableau de Bord Analytique Aérien/src/models/performanceAeroport/RedevanceAeroportGlobalDTO.ts
export class RedevanceAeroportGlobalDTO{
    passagers: number;
    vols :number; 
    montant: number;
    aeroport: string | null;

    constructor(init?: Partial<RedevanceAeroportGlobalDTO>) {
    Object.assign(this, init);
    }   
}